src/test/resources/environment.yaml

```diff
@@ -0,0 +1,101 @@
+version: 1
+# proxy information
+# You only need to configure this, if you plan on white listing proxy boxes and will need to jump
+# through them to talk to the instances that will live in the Cerberus VPC
+# If your not using a proxy either delete this section or make type = DIRECT for direct connecting
+proxy-config:
+  host: localhost
+  port: 9000
+  # [DIRECT, HTTP, SOCKS]
+  type: SOCKS
+
+# The name of the environment
+environment-name: demo
+# The region the environment will run in
+region: us-west-2
+# Costcenter for where to bill provisioned resources. Will be tagged on all resources.
+cost-center: 11111
+# The e-mail for who owns the provisioned resources. Will be tagged on all resources.
+owner-email: obvisouly.fake@nike.com
+# The owning group for the provision resources. Will be tagged on all resources.
+owner-group: cloud platform engineering
+
+# A IAM role ARN that will be given elevated privileges for the KMS CMK created.,
+# If you don't separate root access from admins just use the root role here
+admin-role-arn: arn:aws:iam::111111111:role/onelogin-roles-OneLoginAdminRole-2222222222
+# The Route 53 hosted zone name that will be created for CNAME records used by internal ELBs.
+vpc-hosted-zone-name: demo.internal.cerberus-oss.io
+# The hostname that will be exposed for Cerberus
+hostname: demo.cerberis-oss.io
+# The hostedZoneId that will allow the registration of the hostname
+hosted-zone-id: X5CT6JROG9F2DR
+
+# Here we can define the ports and CIDRs that we want to allow ingress into the Cerberus env VPC
+# You can add your current ip or proxy boxes here and use the above proxy config
+vpc-access-whitelist:
+  ports:
+    - 443
+    - 8080
+    - 8200
+    - 8500
+    - 8400
+    - 22
+  cidrs:
+    - 50.39.106.150/32
+
+consul:

```

Comment:
For each block like this, does it make sense to mention the commands that read from that block in the comments?